packages/core/test/lib/console-child.js

```diff
@@ -0,0 +1,86 @@
+const assert = require("chai").assert;
+const fs = require("fs-extra");
+const path = require("path");
+const tmp = require("tmp");
+const TruffleConfig = require("@truffle/config");
+const { deriveConfigEnvironment } = require("../../lib/command-utils");
+
+let config;
+
+function createSandbox(source) {
+  if (!fs.existsSync(source)) {
+    throw new Error(`Sandbox failed: source: ${source} does not exist`);
+  }
+
+  const tempDir = tmp.dirSync({ unsafeCleanup: true });
+  fs.copySync(source, tempDir.name);
+  const config = TruffleConfig.load(
+    path.join(tempDir.name, "truffle-config.js"),
+    {}
+  );
+  return config;
+}
+
+describe("console-child", function () {
+  before("Create a sandbox", function () {
```

Comment:
Get rid of "Create a sandbox", it is obvious from `createSandbox` that that is what's happening :)

Just omit the string like `before(function () {...`